raiden/ui/cli.py

```diff
@@ -321,7 +333,7 @@ def app(address,
 
     if not os.path.exists(raiden_directory):
         os.makedirs(raiden_directory)
-    user_db_dir = os.path.join(raiden_directory, address[:6])
+    user_db_dir = os.path.join(raiden_directory, address_hex[:6])
```

Comment:
We should probably use `address_hex[:8]` now that the `0x` prefix is included, otherwise we would only be using 2 bytes from the address which would increase the likelihood of a collision.